__tests__/index.test.js

```diff
@@ -224,7 +224,24 @@ describe("Purchases", () => {
     expect(NativeModules.RNPurchases.purchasePackage).toBeCalledWith("$rc_onemonth", "offering", {
       oldSKU: "viejo",
       prorationMode: Purchases.PRORATION_MODE.IMMEDIATE_AND_CHARGE_FULL_PRICE
-    }, null);
+    }, null, null);
+  });
+
+  it("purchaseStoreProduct works", async () => {
```

Comment:
It seems we didn't add a test for `purchaseSubscriptionOption`?

Reply:
Oh, apparently not 😱 I swear I added that... well (re)adding 🤷‍♂️